jetpack-helper.php

```diff
@@ -69,3 +70,19 @@ function jetpack_amp_build_stats_pixel_url() {
 	$data = array_map( 'rawurlencode' , $data );
 	return add_query_arg( $data, 'https://pixel.wp.com/g.gif' );
 }
+
+/**
+ * Force videopress to use html5 player that would generate <video /> tag
+ * that will be later converted to <amp-video />
+ *
+ * @param array $options videopress shortcode options.
+ * @return array videopress shortcode options with `freedom` set to true
+ */
+function amp_videopress_enable_freedom_mode( $options ) {
+	$options['freedom'] = true;
+	return $options;
+}
+
+if ( isset( $_GET[ AMP_Validation_Utils::VALIDATE_QUERY_VAR ] ) ) { // WPCS: CSRF ok.
+	add_filter( 'videopress_shortcode_options', 'amp_videopress_enable_freedom_mode' );
```

Comment:
This shouldn't be necessary now that you have https://github.com/Automattic/amp-wp/pull/1125/commits/b6a4671d12b56a849ccd2d8477b28b68f4f20ef7 right?